pkg/driver/node.go

```diff
@@ -188,9 +189,32 @@ func (ns *node) NodeStageVolume(
 		return nil, status.Error(codes.FailedPrecondition, err.Error())
 	}
 
+	// check the owner node status
+	// if the previous node is not ready, allow
+	// mount to another another node
+	var nodeOk bool
+	if nodeName, ok := instance.Labels["nodeID"]; ok {
+		node, err := ns.client.GetNode(nodeName)
+		if err != nil && !errors.IsNotFound(err) {
+			return nil, status.Error(codes.Internal, err.Error())
+		}
+		if err == nil {
+			nodeOk = isNodeReady(node)
+		}
+	}
+
+	// if the jivaVolume CR has any existing targetPath and the
+	// pevious node is ready abort the stage request
+	if instance.Spec.MountInfo.TargetPath != "" && nodeOk {
+		err = fmt.Errorf("Volume {%v} is already mounted at more than one place: {%v}",
```

Comment:
(nit) (static-check) this will return a minor go tool warning, 
_The error strings shouldn't begin with upper case letter._